mlflow/environment_variables.py

```diff
@@ -186,3 +186,5 @@ def get(self):
 MLFLOW_SCORING_SERVER_REQUEST_TIMEOUT = _EnvironmentVariable(
     "MLFLOW_SCORING_SERVER_REQUEST_TIMEOUT", int, 60
 )
+
+MLFLOW_USE_STDIN_SERVER = _BooleanEnvironmentVariable("MLFLOW_USE_STDIN_SERVER", False)
```

Comment:
We'll set this to `True` in multi-user shared Databricks clusters? Sounds good to me!

Reply:
> We'll set this to True in multi-user shared Databricks clusters?

Yes! I'll add a doc comment.

Reply:
We can confirm this by:

```
> git diff
diff --git a/examples/evaluation/evaluate_on_binary_classifier.py b/examples/evaluation/evaluate_on_binary_classifier.py
index 73d064e10..ab0b588a3 100644
--- a/examples/evaluation/evaluate_on_binary_classifier.py
+++ b/examples/evaluation/evaluate_on_binary_classifier.py
@@ -29,6 +29,7 @@ with mlflow.start_run() as run:
         targets="label",
         model_type="classifier",
         evaluators=["default"],
+        env_manager="virtualenv",
     )

> python examples/evaluation/evaluate_on_binary_classifier.py
> python examples/evaluation/evaluate_on_binary_classifier.py
```